src/model_signing/_cli.py

```diff
@@ -209,7 +209,9 @@ def resolve_command(
 
 
 @click.group(
-    context_settings=dict(help_option_names=["-h", "--help"]),
+    context_settings=dict(
+        help_option_names=["-h", "--help"], max_content_width=100
```

Comment:
Nit: Unsure about the width, since `click` is supposed to get the maximum terminal size when displaying the help.